incubator/nginx-ingress/templates/configmap.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
     k8s-addon: ingress-nginx.addons.k8s.io
 data:
   use-proxy-protocol: "true"
-  custom-http-errors: 404,502,503,504
-
-
+  custom-http-errors: "
+    {{- range $index, $element := .Values.customHttpErrors.client -}}
```

Comment:
I know this looks like madness.
But go template is crap in arrays and lists, that is the best I can do.